tests/ontopy_tests/test_graph2.py

```diff
@@ -1,14 +1,43 @@
 from typing import TYPE_CHECKING
+import owlready2
+import pytest
+from ontopy.graph import OntoGraph
 
 if TYPE_CHECKING:
     from pathlib import Path
 
     from ontopy.ontology import Ontology
 
 
-def test_graph2(emmo: "Ontology", tmpdir: "Path") -> None:
-    from ontopy.graph import OntoGraph
+@pytest.mark.filterwarnings(
+    "ignore:Style not defined for relation hasSpecialRelation."
+)  # currently pytest is set to accept warnings, but this might change in the future
+def test_graph(testonto: "Ontology", tmpdir: "Path") -> None:
+    with testonto:
```

Comment:
```suggestion
def test_graph(testonto: "Ontology", tmpdir: "Path") -> None:
    """Short description of the test.

    Parameters:
        testonto: A local fixture from the root `conftest.py`.
            Returns a local ontology used for testing.
        tmpdir: A built-in pytest fixture to get a function-scoped
            temporary directory.

    """
    with testonto:
```

@jesper-friis has expressed he wants to have doc-strings for tests to explicitly clarify where the fixtures are coming from.